source/extensions/filters/http/ext_authz/ext_authz.h

```diff
@@ -45,8 +45,14 @@ class FilterConfig {
         failure_mode_allow_(config.failure_mode_allow()),
         clear_route_cache_(config.clear_route_cache()),
         max_request_bytes_(config.with_request_body().max_request_bytes()),
-        status_on_error_(toErrorCode(config.status_on_error().code())), local_info_(local_info),
-        scope_(scope), runtime_(runtime), http_context_(http_context), pool_(scope.symbolTable()),
+        status_on_error_(toErrorCode(config.status_on_error().code())),
+        filter_enabled_runtime_key_(config.filter_enabled().runtime_key()),
+        filter_enabled_default_value_(config.has_filter_enabled()
+                                          ? absl::optional<envoy::type::FractionalPercent>(
+                                                config.filter_enabled().default_value())
```

Comment:
Oops sorry disregard this.

Reply:
It is a bit confusing, but the [proto field](https://github.com/envoyproxy/envoy/blob/986173ed516dcc1c3dea7db90659ed993d0aad75/api/envoy/api/v2/core/base.proto#L293) with the real value is called `default_value`.

I don't have a config loading test, but the tests [[1]](https://github.com/gkleiman/envoy/blob/46df1acf0c585d51ac0c0c018393676811f53cc9/test/extensions/filters/http/ext_authz/ext_authz_test.cc#L846-L849) [[2]](https://github.com/gkleiman/envoy/blob/46df1acf0c585d51ac0c0c018393676811f53cc9/test/extensions/filters/http/ext_authz/ext_authz_test.cc#L872-L875) I added do verify that the runtime loader is called with the right value.

Writing a config loading test might be a bit tricky in this case, because the config values are stored in private fields and not directly exposed.

Please re-open this if I you still think I'm using the wrong value or if I haven't convinced you that the current tests are enough =).

Reply:
Sorry see above, disregard my original comment. This is correct.